src/flows/context/flow.current.tsx

```diff
@@ -69,37 +69,45 @@ export const FlowProvider: FC = ({children}) => {
   }, [flows, currentID])
 
   useEffect(() => {
-    if (isFlagEnabled('sharedFlowEditing')) {
+    const doc = yDoc.current
+    if (isFlagEnabled('sharedFlowEditing') && currentID) {
       provider.current = new WebsocketProvider(
         'wss://demos.yjs.dev', // todo(ariel): replace this with an actual API that we setup
-        flowId, // todo(ariel): we might need to confine this to an org, depends on how multi-org plays out
-        yDoc.current
+        currentID, // todo(ariel): we might need to confine this to an org, depends on how multi-org plays out
```

Comment:
re org: nah. notebook id should be unique across orgs

Reply:
Totally